### PR TITLE
Add response for success response

### PR DESCRIPTION
**Pull request by berenda-andrey**
It is my fault. Before I changed it

### CODE REVIEW

File: tests/schema/test_generation.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    class SimpleController:
+
+        @winter.route_get('/simple/')
+        def simple_method(self) -> return_type:
+            return None
+
+    route = get_route(SimpleController.simple_method)
+
+    # Act
+    schema = build_response_schema(route)
```
Comment (mofr) [inline tests/schema/test_generation.py]: The function is already tested in `tests/schema/test_response_schema.py`. Please merge these two files.

File: winter/schema/generation.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from drf_yasg import openapi
 from drf_yasg.utils import swagger_auto_schema
 
+from winter.core import ComponentMethod
```
Comment (mofr) [inline winter/schema/generation.py]: Maybe convert to a relative import?

File: winter/pagination/page_position_argument_inspector.py [hunk 3]
```diff
@@ -25,9 +24,9 @@ def __init__(self, page_position_argument_resolver: PagePositionArgumentResolver
             type=openapi.TYPE_INTEGER,
         )
 
-    def inspect_parameters(self, method: ComponentMethod) -> List[openapi.Parameter]:
+    def inspect_parameters(self, route: 'Route') -> List[openapi.Parameter]:
```
Comment (berenda-andrey) [inline winter/pagination/page_position_argument_inspector.py]: it is also my fault

File: winter/exceptions.py [hunk 4]
```diff
@@ -50,15 +50,8 @@ class ExceptionHandler(abc.ABC):
...
-        from .routing import Route
-
         handle = self.__class__.handle
-        handle = annotate(None)(handle)
-        self.route = Route(
-            http_method='',
-            url_path='',
-            method=handle
-        )
+        self.handle_method = annotate(None)(handle)
```
Comment (mofr) [inline winter/exceptions.py]: Why not `Component.register`?

**Pull request merged**